zebra/rt_netlink.c

```diff
@@ -1535,6 +1535,12 @@ static int netlink_route_multipath(int cmd, struct zebra_dplane_ctx *ctx)
 		char buf[NL_PKT_BUF_SIZE];
 	} req;
 
+	if (zrouter.skip_kernel_install) {
```

Comment:
Another question: why choose this api, rather than `kernel_route_update`?

Reply:
Yes i agree with you and i have already comment a proposal for this.